storm-server/src/main/java/org/apache/storm/scheduler/blacklist/BlacklistScheduler.java

```diff
@@ -89,13 +89,8 @@ public void prepare(Map<String, Object> conf) {
         cachedSupervisors = new HashMap<>();
         blacklistHost = new HashSet<>();
 
-        StormMetricsRegistry.registerGauge("nimbus:num-blacklisted-supervisor", new Callable<Integer>() {
-            @Override
-            public Integer call() throws Exception {
-                //nimbus:num-blacklisted-supervisor + none blacklisted supervisor = nimbus:num-supervisors
-                return blacklistHost.size();
-            }
-        });
+        //nimbus:num-blacklisted-supervisor + none blacklisted supervisor = nimbus:num-supervisors
```

Comment:
nit: I have no idea what this comment means, so could we remove it?  Unless you understand it, then can we make it clearer?

Reply:
It looks to me that there's a typo: nimbus:num-blacklisted-supervisor + non-blacklisted supervisor = nimbus:num-supervisors